### ISSUE TITLE
Issues when building Player (CMake)

### ISSUE THREAD

**Issue by funky327** [SudoMaker/RetroWave#1]
While attempting to build on macOS, I get the following error:

```
cmake ..; make
-- Omg: you are using clang!
-- Note: Building the player by default. This requires additional dependencies. Use -DRETROWAVE_BUILD_PLAYER=0 to disable.
-- CPM: adding package cxxopts@3.0.0 (df229cff0d5b96e146f3f11441f714e8e240cad0)
[ 11%] Performing download step (git clone) for 'cxxopts-populate'
Cloning into 'cxxopts-src'...
fatal: reference is not a tree: df229cff0d5b96e146f3f11441f714e8e240cad0
CMake Error at cxxopts-subbuild/cxxopts-populate-prefix/tmp/cxxopts-populate-gitclone.cmake:40 (message):
  Failed to checkout tag: 'df229cff0d5b96e146f3f11441f714e8e240cad0'


make[2]: *** [cxxopts-populate-prefix/src/cxxopts-populate-stamp/cxxopts-populate-download] Error 1
make[1]: *** [CMakeFiles/cxxopts-populate.dir/all] Error 2
make: *** [all] Error 2
```

Thanks!

**Comment by lethal-guitar** [comment]
Same problem here. It works for me when I apply the following diff:

```
diff --git a/CMakeLists.txt b/CMakeLists.txt
index f06620e..c2ab372 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,7 +46,7 @@ if(${RETROWAVE_BUILD_PLAYER} EQUAL 1)
             NAME cxxopts
             GITHUB_REPOSITORY jarro2783/cxxopts
             VERSION 3.0.0
-            GIT_TAG df229cff0d5b96e146f3f11441f714e8e240cad0
+            GIT_TAG c74846a891b3cc3bfa992d588b1295f528d43039
             GIT_SHALLOW ON
     )
 
```

**Comment by ReimuNotMoe** [comment]
Sorry for the inconvenience. I'll fix it.

**Comment by ReimuNotMoe** [comment]
Fixed in cedbcce

**Comment by lethal-guitar** [comment]
@ReimuNotMoe thanks for the quick response! 👍🏻
